### PR TITLE
Genarlise env.py to get database URL from .env Fixes: #1043

### PR DESCRIPTION
**Pull request by Zaffer**
### Description
it is better to get your private database url from an .env file. Putting secrets in your ini file is not the best, is it?

### Checklist
This pull request is:

- [ ] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [ ] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [x] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

File: alembic/templates/generic/env.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
+import os
 from logging.config import fileConfig
 
+from dotenv import load_dotenv
```
Comment (CaselIT) [inline alembic/templates/generic/env.py:4]: I don't think the general template should import a 3rd party dependency

File: alembic/templates/generic/env.py [hunk 2]
```diff
@@ -25,6 +30,12 @@
 # my_important_option = config.get_main_option("my_important_option")
 # ... etc.
 
+alembic_url = os.getenv(
+    "ALEMBIC_URL", "postgresql+psycopg2://postgres:postgres@localhost:5432/postgres"
```
Comment (CaselIT) [inline alembic/templates/generic/env.py:34]: The general template should not assume any url

**Review by CaselIT — CHANGES_REQUESTED** [review]
I think it may make sense to use the env variable `ALEMBIC_URL` for the url and if not found fall-back on the ini file, but the env file should not assume 3rd party packages or a particular db.

**Comment by zzzeek** [comment]
hi -

env.py files are templates that you can modify in any way you see fit, including using your own configuration systems.    the template provided is a generic example only.

File: alembic/templates/generic/env.py (hunk 1 above)
Comment (Zaffer) [reply alembic/templates/generic/env.py:4]: fair

File: alembic/templates/generic/env.py (hunk 2 above)
Comment (Zaffer) [reply alembic/templates/generic/env.py:34]: got it

**Pull request closed without merging**